js/background.js

```diff
@@ -14,6 +14,7 @@ define(function(require) {
 	'use strict';
 
 	var OC = require('OC');
+	var BackGround = this;
```

Comment:
Seems like travis failed because you are not using _this(?)
This might be nitpicky but should background be written without the camelCase since it's just one word?

Also the scrutinizer lists 11 code style issues but they were there before too. Intentional?